src/dex/aave-v1/aave-v1-integration.test.ts

```diff
@@ -24,11 +25,7 @@ describe('AaveV1', function () {
       return;
     }
 
-    const amounts = [
-      BigInt('0'),
-      BigInt('1000000000000000000'),
-      BigInt('2000000000000000000'),
-    ];
+    const amounts = [BI_0, BI_POWS[18], BigInt('2000000000000000000')];
```

Comment:
I see we use `BigInt('2000000000000000000')` quite often. Can we constant it too ?

Reply:
They are all in tests, so I didn't want to inflate the consts, but I currently repaced all of them with `n` postfixed values